src/main/java/org/mskcc/cbio/oncokb/config/CacheConfiguration.java

```diff
@@ -36,8 +36,8 @@
     private final String PROP_NAME = "enabled";
 
     @Bean
-    @ConditionalOnProperty(prefix = PROP_PREFIX, name = PROP_NAME, havingValue = "false")
-    public org.springframework.cache.CacheManager cacheManager(ApplicationProperties applicationProperties){
+    @ConditionalOnProperty(prefix = PROP_PREFIX, name = PROP_NAME, havingValue = "false", matchIfMissing = true)
```

Comment:
Conditions are evaluated before ConfigurationProperties properties are available in Spring, so when "application.redis.enabled" is not present in yml file, then the custom cache manager is not created. matchIfMissing sets the default behavior when that property is missing.